docs/developer-docs/latest/setup-deployment-guides/configurations/databases/sqlite.md

```diff
@@ -34,6 +34,40 @@ This will create a new project and launch it in the browser.
 The [Quick Start Guide](/developer-docs/latest/getting-started/quick-start.md) is a complete step-by-step tutorial.
 :::
 
+## Install SQLite manually
+
+In terminal, run the following command:
+
+<code-group>
+
+<code-block title="NPM">
+```sh
+npm install better-sqlite3
+```
+</code-block>
+
+<code-block title="YARN">
+```sh
+yarn install better-sqlite3
+```
+</code-block>
+
+</code-group>
+
+Add the following to your `./config/database.js` file:
+
+```js
+module.exports = ({ env }) => ({
+  connection: {
+    client: 'sqlite',
+    connection: {
+      filename: path.join(__dirname, '..', env('DATABASE_FILENAME', '.tmp/data.db')),
+    },
+    useNullAsDefault: true,
+  },
+});
+```
+
```

Comment:
Can you add the following code for TypeScript as well (using the code-group/code block like above)?

```
import path from 'path';

export default ({ env }) => ({
  connection: {
    client: 'sqlite',
    connection: {
      filename: path.join(__dirname, '..', '..', env('DATABASE_FILENAME', '.tmp/data.db')),
    },
    useNullAsDefault: true,
  },
});
```